pages/accountLists/[accountListId]/contacts/[[...contactId]].page.tsx

```diff
@@ -47,12 +47,22 @@ const ContactsPage: React.FC = () => {
           }
         : {
             pathname: '/accountLists/[accountListId]/contacts/',
-            query: queryWithoutContactId,
+            query: { ...queryWithoutContactId },
           },
     );
     setContactDetailsId(id);
   };
 
+  const setSearchTerm = (searchTerm?: string) => {
+    push({
```

Comment:
Do we want `push` or `replace`? Is this updating on every character press?

Reply:
Yeah, it updates on every character press, but it's using the debounced input.

Reply:
Maybe ```replace``` would be better so the history isn't filled up?

Reply:
👍 I was talking with Spencer about contact filters and I think we want to eventually just store them in the url but don't want to clutter the history. So ya I think replace is a good choice.